operator-framework-core/pom.xml

```diff
@@ -56,6 +56,7 @@
     <dependency>
       <groupId>io.fabric8</groupId>
       <artifactId>openshift-client</artifactId>
+      <version>${fabric8-client.version}</version>
```

Comment:
I don't know why, Maven is complaining about those missing versions ... any hint on how to solve the issue without this change? cc. @csviri

Reply:
Hmm, actually, it is so not sure why it's complaining now when it wasn't before… :(

Reply:
could it be related to a local setup? I think I don't have anything fancy in place ...

Reply:
Fixed, for the records, I had cached a "bad version" of the `kubernetes-client` BOM.